app-backend/tile/src/main/resources/application.conf

```diff
@@ -7,3 +7,37 @@ tile-server {
   bucket = "rasterfoundry-staging-catalogs-us-east-1"
   bucket = ${?TILE_SERVER_BUCKET}
 }
+
+kamon {
+  metric {
+    filters {
+      trace.includes = [ "**" ]
+    }
+  }
+
+  subscriptions {
+    trace                = [ "**" ]
+    trace-segment        = [ "**" ]
+    akka-http-server     = [ "**" ]
+  }
+
+  statsd {
+    hostname = "statsd"
+    port = 8125
+    time-units = "ms"
+    simple-metric-key-generator.include-hostname = false
```

Comment:
Remove the `host` from the metrics namespace. Otherwise, we end up with new metrics every time a service cycles in/out.